### DISCUSSION TITLE
add staging environment option

### DISCUSSION THREAD

**Discussion by Snuupy** [ZoeyVid/NPMplus#1082] [Q&A] [answered]
I'd like to use the letsencrypt staging environment before I deploy the new reverse proxy.

In NPM, it's `LE_STAGING: 'true'` it looks like https://github.com/NginxProxyManager/nginx-proxy-manager/blob/63d06da8a8591e7a9b2a1873eb91ce1c42b2b0f9/docker/docker-compose.dev.yml#L23

but the same implementation is not available in NPMplus: https://github.com/NginxProxyManager/nginx-proxy-manager/blob/63d06da8a8591e7a9b2a1873eb91ce1c42b2b0f9/backend/internal/certificate.js#L973 exists in NPM but in NPMplus there is no staging option. https://github.com/ZoeyVid/NPMplus/blob/develop/backend/internal/certificate.js

**Comment by Zoey2936** [comment, marked as answer]
Hello, you can edit the certbot acme servers in the config file

**Reply by Zoey2936** [reply]
(/opt/npm/tls/certbot/config.ini)

**Reply by Snuupy** [reply]
thanks! I will take a look.
